drivers/hid/hid-sony.c

```diff
@@ -1067,8 +1067,24 @@ static int sixaxis_set_operational_usb(struct hid_device *hdev)
 static int sixaxis_set_operational_bt(struct hid_device *hdev)
 {
 	unsigned char buf[] = { 0xf4,  0x42, 0x03, 0x00, 0x00 };
-	return hid_hw_raw_request(hdev, buf[0], buf, sizeof(buf),
-				  HID_FEATURE_REPORT, HID_REQ_SET_REPORT);
+	unsigned char ack;
+	ack = hid_hw_raw_request(hdev, buf[0], buf, sizeof(buf), HID_FEATURE_REPORT, HID_REQ_SET_REPORT);
+	unsigned char enable_gasia[] = {
+			0xA2,
+			0x01,
+			0x00, 0x00, 0x00, 0x00, 0x00,   // rumble values [0x00, right-timeout, right-force, left-timeout, left-force]
+			0x00, 0x00, 0x00, 0x00, 0x02,   // 0x02=LED1 .. 0x10=LED4
+			0xff, 0x27, 0x10, 0x00, 0x32,   // LED 4
+			0xff, 0x27, 0x10, 0x00, 0x32,   // LED 3
+			0xff, 0x27, 0x10, 0x00, 0x32,   // LED 2
+			0xff, 0x27, 0x10, 0x00, 0x32,   // LED 1
+			0x00, 0x00, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00
+	};
+	hid_hw_raw_request(hdev, enable_gasia[0], enable_gasia, sizeof(enable_gasia), HID_FEATURE_REPORT, HID_REQ_SET_REPORT);
```

Comment:
What happens if we send this request to a controller that does not support it? Do we get a bad `ack` value?